### DISCUSSION TITLE
Error in DWIN.CPP Line 2882

### DISCUSSION THREAD

**Discussion by Crazy-Charles** [mriscoc/Ender3V2S1#945] [General]
I know its not a common setting but as I have added a caselight to my board and firmware, this line fails to compile...(same in previous versions but never got around to mentioning it!)

MENU_ITEM(ICON_CaseLight, MSG_CASE_LIGHT, onDrawChkbMenu, SetCaseLight, &caselight.on);

Needs changing to EDIT_ITEM.

**Comment by mriscoc** [comment]
You are right, thanks.
